### ISSUE TITLE
Generated client contains wrong typings for "fee" parameter

### ISSUE THREAD

**Issue by giovanni-orciuolo** [pyramation/cosmwasm-typescript-gen#9]
Hello, first of all thanks for the package, it's so great and easy to use!

I'm trying to generate the Query/Execute clients starting from my own schema, but I'm noticing that all methods which contain the "fee" parameter have these typings:
```
fee: number | StdFee = "auto"
```

When it should be:
```
fee: number | StdFee | "auto" = "auto"
```

The fact that the default value "auto" is not included in the types possibilities makes TS angry. I could fix each one by hand, but then if I regenerate I get the same problem.

**Comment by pyramation** [comment]
Thanks for finding this! will update shortly @giovanni-orciuolo

**Comment by pyramation** [comment]
should be fixed in `cosmwasm-typescript-gen@0.2.3`
